src/test/HelmetTest.jsx

```diff
@@ -609,5 +611,22 @@ describe("Helmet", () => {
             expect(existingTag.getAttribute("content")).to.equal("This is \"quoted\" text and & and '.");
             expect(existingTag.outerHTML).to.equal(`<meta name="description" content="This is &quot;quoted&quot; text and &amp; and '." ${HELMET_ATTRIBUTE}="true">`);
         });
+
+        it("will html encode title on server", () => {
+            ExecutionEnvironment.canUseDOM = false;
```

Comment:
Reluctantly did this hack as the test scripts are only running on the browser currently.